packages/components/src/HeaderBar/HeaderBar.test.js

```diff
@@ -4,7 +4,7 @@ import HeaderBarComponent from './HeaderBar.component';
 
 describe('HeaderBar', () => {
 	it('should render', () => {
-		const wrapper = mount(<HeaderBarComponent.WrappedComponent />);
+		const wrapper = mount(<HeaderBarComponent />);
```

Comment:
Talend/scripts i18n HOC mock returns the original component. No WrappedComponent anymore